client/__tests__/util/stateManager/summarizeAnnotations.test.js

```diff
@@ -69,18 +75,27 @@ describe("summarizeAnnotations", () => {
   });
 
   test("simple test", () => {
-    const obsAnnotations = [
-      {
-        __index__: 0,
-        name: "n1",
-        nameString: "hi",
-        nameBoolean: true,
-        nameFloat32: 39.3,
-        nameInt32: 99,
-        nameCategorical: 1
-      }
-    ];
-    const varAnnotations = [];
+    const obsAnnotations = new Dataframe.Dataframe(
```

Comment:
I recently learned about the faker.js package for creating data for tests, might be useful for cases like this. (Not recommending it for this particular PR, though)